types/api/category.d.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+type CategorySorting = '' | 'alpha' | 'manual' | 'recent'
```

Comment:
does empty mean, no sort?

Reply:
Empty / default is `recent`.

Reply:
we have two values meaning the same? that seems weird! we should only have valid values in the type, and have the defaults were needed.

Reply:
I've removed the empty string and made it so that the transformer sets the value as `recent` if the api value is not set or empty